Core/Systems/Questing/QuestModPlayer.cs

```diff
@@ -1,29 +1,21 @@
-﻿using PathOfTerraria.Core.Systems.Affixes;
-using PathOfTerraria.Core.Systems.Questing.Quests.TestQuest;
-using PathOfTerraria.Core.Systems.Questing.QuestStepTypes;
-using System;
+﻿using PathOfTerraria.Core.Systems.Questing.Quests.TestQuest;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using System.Xml.Linq;
-using Terraria.GameContent.UI.Chat;
 using Terraria.ModLoader.IO;
 
 namespace PathOfTerraria.Core.Systems.Questing;
-internal class QuestHandler : ModPlayer
+internal class QuestModPlayer : ModPlayer
 {
 	// need a list of what npcs start what quests
-	private readonly List<Quest> _enabledQuests = [];
+	public readonly List<Quest> EnabledQuests = [];
```

Comment:
Shouldn't this rather be private with methods to get the completed and in-progress quests?

Reply:
Added functions to do this, however I did keep it public because I feel we may want those.

Reply:
I'd personally aim for making it internal if not private, in the off chance we make noticeable cross mod content it'd be a little safer for those users.

Reply:
QuestModPlayer is already internal

Reply:
Ah, alright then. No issues from me there.

Reply:
Made it private, we can always mark as public if we want the full list later